packages/parcel-transformer/src/index.ts

```diff
@@ -127,8 +132,7 @@ export default new Transformer<ParcelTransformerOpts>({
         config.extract && [
           '@compiled/babel-plugin-strip-runtime',
           {
-            styleSheetPath: 'compiled-css!',
-            compiledRequireExclude: config.ssr,
+            compiledRequireExclude: true,
```

Comment:
I am thinking of making a breaking change to the parcel packages as splitting css every async js split defeats the purpose of atomic css. It means 
- deprecate parcel-resolver 
- major bump parcel-transformer
- use parcel-optimizer to insert css to HTML, and provide an option to inline/externalise css.

Reply:
had a discussion with Jake,
We'll
Deprecate parcel-resolver
Update parcel-config
Rename 'parcel-optimizer-html-inline-css' to parcel-optimiser